test/statschol.jl

```diff
@@ -1,16 +1,14 @@
-using LinearAlgebra, MixedModels, StatsModels, Random, Test
+using LinearAlgebra, MixedModels, StatsModels, Test
 
 xtx(X) = X'X  # creat the symmetric matrix X'X from X
 
-const rng = Random.MersenneTwister(4321234)
-
 const simdat = (
     G = repeat('A':'T', inner=10),
     H = repeat('a':'e', inner=2, outer=20),
-    U = repeat(0.:9, outer=20), 
+    U = repeat(0.:9, outer=20),
     V = repeat(-4.5:4.5, outer=20),
-    Y = 0.1 * randn(rng, 200),
-    Z = rand(rng, 200)
+    Y = (1:200)/200,
```

Comment:
Random data seems better in general than a regular range. Any reason to change this?

Reply:
@nalimilan for both of these issues, the change in RNG in Julia 1.5 breaks things (see #306). I prefer the old tests too.  :man_shrugging: 

(That said, I'm going to see about using hard-coded RNG values that I generate once on my local machine as the next step in looking at the Cholesky issue #324.()

Reply:
Yes, that's annoying, but how about using StableRNGs? (I'm surprised the test is so sensitive to changing the exact values. Maybe that's just bad luck?)

Reply:
Yeah, it's really bad luck. Also, StableRNGs is partly the result of the fit I threw when I was told about this change. ;)

Reply:
Back to using (pseudo)0randomness.